crates/matrix-sdk-crypto/src/store/error.rs

```diff
@@ -77,8 +77,14 @@ pub enum CryptoStoreError {
 
     /// A problem with the underlying database backend
     #[error(transparent)]
+    #[cfg(not(target_family = "wasm"))]
     Backend(Box<dyn std::error::Error + Send + Sync>),
 
+    /// A problem with the underlying database backend
+    #[error(transparent)]
+    #[cfg(target_family = "wasm")]
+    Backend(Box<dyn std::error::Error>),
```

Comment:
@zzorba I'm just wondering what the driver for this particular change (removing the `Send + Sync` bounds on `Backend`) was?

(For context: it turns out we have some code in matrix-sdk-crypto-wasm that relies on `CryptoStoreError` being `Send + Sync`. Not the end of the world to work around it, but I'm interested to know why it was necessary.)

Reply:
Okay, so this particular one is not needed.  

One of them is, the `UnknownError` in matrix-sdk/src/error.rs runs into anyhow, which struggles with the sync bounds.  

I will open a PR to relax the ones I can though.